fab/fabfile.py

```diff
@@ -489,38 +494,35 @@ def development():
     env.django_port = '9010'
     env.should_migrate = True
 
-    require('inventory')
-
-    # use inventory filename as environment name
-    # i.e. if the inventory is called my-crazy-setup
-    # then things on the server will be stored in
-    # /home/cchq/www/my-crazy-setup/code_root, etc.
-    env.environment = os.path.basename(env.inventory)
+    require('inventory', 'environment')
```

Comment:
I've actually been using this command to deploy to india. Once it's solidified, we'll probably want to make a special target for india (or rather edit the previous target), but there's also something nice about having it go directly off the inventory file we're also using for ansible, so I'm not 100% sure what I'll do, but I'm not going to stick with this